src/components/Home/Home.js

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCountries, navigation } from '../../redux/countries/countrySlice';
+import Countries from '../Countries/Countries';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state.countries);
+  const { countries } = data;
+  useEffect(() => { dispatch(fetchCountries()); dispatch(navigation('home')); }, [dispatch]);
+  let totalPopulation = 0;
+  const continents = [];
+  // Use data retrieve stats
+  if (countries.length) {
+    continents.push({ continent: countries[0].continents[0], population: 0 });
+    countries.forEach((country) => {
+      totalPopulation += country.population;
+      if (!continents.some((i) => i.continent === country.continents[0])) {
+        continents.push({ continent: country.continents[0], population: 0 });
+      }
+      if (continents.some((i) => i.continent === country.continents[0])) {
+        continents.forEach((i) => {
+          if (i.continent === country.continents[0]) {
+            continents[continents.indexOf(i)].population += country.population;
+          }
+        });
+      }
+    });
+  }
+  return (
+    <>
+      <div className="App__home">
+        {countries.length !== 0 && (
+        <>
+          <div className="App__home--heading overlay">
+            <h1 className="text">World Population</h1>
+            {' '}
+            <span className="text" style={{ fontSize: '1em' }}>{totalPopulation.toLocaleString('en-US')}</span>
+          </div>
+          <ul className={!data.search ? 'App__grid-container p-0' : 'd-none'}>
+            {continents.map((continent) => (
+              <NavLink key={continent.continent} to={`/${continent.continent}`}>
+                <li className="App__home-region d-flex flex-column justify-content-end align-items-end px-2 py-3 overlay">
+                  <h2>{continent.continent}</h2>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233610820-95752674-3f85-487a-b693-a1c035bdf0c6.png)

- [ ] Kudos on implementing some of the changes requested by the previous reviewer :+1: but it would be nice if you should make your continent names in UPPERCASE to follow the design guidelines which is also mentioned by the previous reviewer :thinking: